server/asset/build/btc/btc.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
+	"decred.org/dcrdex/server/asset"
+	"decred.org/dcrdex/server/asset/btc"
+)
+
+func init() {
+	asset.Register(btc.AssetName, &btc.Driver{})
+}
```

Comment:
Plugin backends will need to `Register` themselves, so I think that's the pattern we should use here too. Let's move these back to the **server/asset/{btc,ltc.dcr}** package `init`s.

Reply:
> Plugin backends will need to `Register` themselves, so I think that's the pattern we should use here too. Let's move these back to the **server/asset/{btc,ltc.dcr}** package `init`s.

fixed